### PR TITLE
Add licence plate number to new report

### PR DESCRIPTION
**Pull request by mjurkus**
https://trello.com/c/NFgYvkhu/175-add-plate-number-field-when-registering-new-parking-violation-report

### CODE REVIEW

File: app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ReportRegistrationActivity : BaseActivity() {
                 .beginTransaction()
                 .replace(android.R.id.content, PhotoInstructionsFragment.newInstance())
                 .addToBackStack(null)
-                .commit()
+                .commitAllowingStateLoss()
```
Comment (vycius) [inline app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt:34]: Why ?

File: app/src/main/res/layout/fragment_new_report.xml [hunk 2]
```diff
@@ -85,6 +85,33 @@
...
+                        android:layout_height="wrap_content"
+                        android:paddingEnd="24dp"
+                        android:paddingRight="24dp">
+
+                        <android.support.design.widget.TextInputEditText
+                            android:id="@+id/report_problem_licence_plate_number"
+                            style="@style/NewProblemField"
+                            android:hint="@string/hint_new_report_licence_plate"
+                            android:inputType="textCapCharacters"
+                            android:maxLength="6"
```
Comment (vycius) [inline app/src/main/res/layout/fragment_new_report.xml]: Just for Lithuanian plate numbers ?

**Review by vycius — APPROVED** [review]

File: app/src/main/res/layout/fragment_new_report.xml (hunk 2 above)
Comment (mjurkus) [reply app/src/main/res/layout/fragment_new_report.xml]: Yeah, have to remove...

File: app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt (hunk 1 above)
Comment (mjurkus) [reply app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt:34]: Hmm, forgot to test that...
There was a crash when asking for camera permissions, after approval it tries to display photo instructions and crashes.

Comment (vycius) [reply app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt:34]: How this change will fix crash ?

Comment (mjurkus) [reply app/src/main/java/lt/vilnius/tvarkau/activity/ReportRegistrationActivity.kt:34]: This allows to commit fragment even after activity `onSaveInstanceState`

**Pull request merged**